docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+nightscout:
+  build: .
+  links:
+   - database
+   - broker
+  ports:
+   - "1337:1337"
+  environment:
+   - MONGO_CONNECTION=mongodb://database/nightscout
+   - API_SECRET=mylittlesecret
```

Comment:
Good point Jason, I have commented the line out, so people have to explicitly set the secret to enable the API.